### PR TITLE
Add OSTree packages to comps

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

**Comment by parthaa** [comment]
Also need to tag 
pulp-ostree-plugins and pulp-ostree-admin-extensions

File: comps/comps-katello-pulp-server-rhel6.xml [hunk 1]
```diff
@@ -27,7 +27,6 @@
        <packagereq type="default">pulp-nodes-child</packagereq>
        <packagereq type="default">pulp-nodes-common</packagereq>
        <packagereq type="default">pulp-nodes-parent</packagereq>
-       <packagereq type="default">pulp-ostree</packagereq>
```
Comment (ehelms) [inline comps/comps-katello-pulp-server-rhel6.xml:30]: We aren't shipping ostree on EL6 bits - is that correct?

Comment (parthaa) [inline comps/comps-katello-pulp-server-rhel6.xml:30]: no plans right now

**Comment by jlsherrill** [comment]
ACK from me

**Comment by ehelms** [comment]
I was only holding merging until we tagged them over into our tags as a reminder (so hopefully you did that or are planning to :))

**Comment by jlsherrill** [comment]
@ehelms its there:  http://koji.katello.org/koji/buildinfo?buildID=28677

**Pull request merged**